ci/requirements-py36.yml

```diff
@@ -22,6 +22,9 @@ dependencies:
   - zarr
   - pseudonetcdf>=3.0.1
   - eccodes
+  # - cdat-lite  # No py3 support
+  - pynio
+  - iris>=1.10
```

Comment:
neither pynio or iris have a py37 build yet. cdat-lite is py2 only.

Reply:
It looks like we can get just the `cdms2` module that xarray needs from conda-forge (which does have builds for Python 3.6).